robots.txt

```diff
@@ -0,0 +1,2 @@
+User-agent: *
```

Comment:
dont see how this file would get used anywhere, I think you'd need to put it into the 'static' folder for it to get copied over